src/components/RouteWrapper.tsx

```diff
@@ -284,6 +285,13 @@ export const ROUTES: RouteProperties[] = [
         description: "Arjun can find query parameters for URL endpoints.",
         category: "Web Application Testing",
     },
+    {
+        name: "Arpaname",
+        path: "/tools/Arpaname",
+        element: <ArpanameTool />,
+        description: "Perform reverse DNS lookups for IP addresses.",
```

Comment:
Please make the description consistent with the description constant in arpaname.tsx.